### DISCUSSION TITLE
How do we remove Trial Mode?

### DISCUSSION THREAD

**Discussion by christoferd** [statamic/cms#10841] [Troubleshooting] [answered]
How do we remove Trial Mode?
How can I set the CMS to "Free Version"?

I'm on my first install on my dev machine with a "Free" starter kit from the statamic site "studio1902/statamic-peak"
I created a user, using the CLI. Using php artisan serve to run it on http://127.0.0.1:8000
... which I changed in the .env file too.
Now it says Trial Mode ... why is that?
Click on "Learn more..." only for the info page to tell me that I'm using something from Pro version. 
What could that be? Who knows? The app doesn't explain it!

**All I want is to use the basic free version and try it out. I just want to see if it works the way I need it to.** 

Please provide refs to documentation so I can learn how to get around these issues, or install a fresh version that is just the free version.

PS: Trying Statamic just to see if this is a good fit for my needs.

**Comment by ryanmitchell** [comment, marked as answer]
If you want to switch back to the free version, ensure that `pro` is set to `false` in `config/statamic/editions.php`
